test/WebSites/ValidationWebSite/CompanyNameAttribute.cs

```diff
@@ -3,19 +3,16 @@
 
 using System.ComponentModel.DataAnnotations;
 
-namespace FormatterWebSite
+namespace ValidationWebSite
 {
-    public class AdminValidator : ValidationAttribute
+    public class CompanyNameAttribute : ValidationAttribute
     {
         protected override ValidationResult IsValid(object value, ValidationContext validationContext)
         {
-            var admin = (Administrator)value;
-            
-            if (admin.AdminAccessCode != 1)
+            if (value == null)
```

Comment:
This logs a validation error when value is null. I cannot ensure it is a string when value is null right? For cases where value is not null, I am anyways not doing anything and returning null.